File: DabaseAndroidAppJava/MainActivity.java
package com.example.admin.mydatabase;

import android.database.Cursor;
import android.database.sqlite.SQLiteDatabase;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView lst;
    EditText studentid;
    EditText studentname;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        lst = (TextView) findViewById(R.id.list);
        studentid = (EditText) findViewById(R.id.studentID);
        studentname = (EditText) findViewById(R.id.studentName);

    }
    public void addStudent (View view) {
        MyDBHandler dbHandler = new MyDBHandler(this, null, null, 1);
        int id = Integer.parseInt(studentid.getText().toString());
        String name = studentname.getText().toString();
        Student student = new Student(id,name);
        dbHandler.addHandler(student);
        studentid.setText("");
        studentname.setText("");
    }

    public void findStudent (View view) {
        MyDBHandler dbHandler = new MyDBHandler(this, null, null, 1);
        Student student = dbHandler.findHandler(studentname.getText().toString());
        if (student != null) {
            lst.setText(String.valueOf(student.getID()) +" "+ student.getStudentName());

        } else {
            lst.setText("No Match Found");
        }
    }

    public void loadStudents(View view) {
        MyDBHandler dbHandler = new MyDBHandler(this, null, null, 1);
        lst.setText(dbHandler.loadHandler());
        studentid.setText("");
        studentname.setText("");
    }

    public void deleteStudent(View view) {
        MyDBHandler dbHandler = new MyDBHandler(this, null,
                null, 1);
        boolean result = dbHandler.deleteHandler(Integer.parseInt(
                studentid.getText().toString()));
        if (result) {
            studentid.setText("");
            studentname.setText("");
            lst.setText("Record Deleted");
        } else
            studentid.setText("No Match Found");
    }

    public void updateStudent(View view) {
        MyDBHandler dbHandler = new MyDBHandler(this, null,
                null, 1);
        boolean result = dbHandler.updateHandler(Integer.parseInt(
                studentid.getText().toString()), studentname.getText().toString());
        if (result) {
            studentid.setText("");
            studentname.setText("");
            lst.setText("Record Updated");
        } else
            studentid.setText("No Match Found");
    }
}

File: DatabaseAndroidAppKotlin/MainActivity.kt
package com.example.admin.mydatabasekotlinapp

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.example.admin.mydatabasekotlinapp.R.id.studentname
import com.example.admin.mydatabasekotlinapp.R.id.studentid
import com.example.admin.mydatabasekotlinapp.R.id.lst
import kotlinx.android.synthetic.main.activity_main.*
import com.example.admin.mydatabasekotlinapp.R.id.studentname
import com.example.admin.mydatabasekotlinapp.R.id.studentid
import com.example.admin.mydatabasekotlinapp.R.id.studentname
import com.example.admin.mydatabasekotlinapp.R.id.studentid
import com.example.admin.mydatabasekotlinapp.R.id.lst
import com.example.admin.mydatabasekotlinapp.R.id.studentid
import com.example.admin.mydatabasekotlinapp.R.id.lst
import com.example.admin.mydatabasekotlinapp.R.id.studentname
import com.example.admin.mydatabasekotlinapp.R.id.studentid
import com.example.admin.mydatabasekotlinapp.R.id.lst
import com.example.admin.mydatabasekotlinapp.R.id.studentname




//import sun.rmi.registry.RegistryImpl.getID




class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun loadStudents(view: View) {
        val dbHandler = MyDBHandler(this, null, null, 1)
        lst.setText(dbHandler.loadHandler())
        studentid.setText("")
        studentname.setText("")
    }

    fun addStudent(view: View) {
        val dbHandler = MyDBHandler(this, null, null, 1)
        val id = Integer.parseInt(studentid.text.toString())
        val name = studentname.text.toString()
        val student = Student(id, name)
        dbHandler.addHandler(student)
        studentid.setText("")
        studentname.setText("")
    }

    fun findStudent(view: View) {
        val dbHandler = MyDBHandler(this, null, null, 1)
        var student = dbHandler.findHandler(studentname.text.toString())
        if (student != null) {
            lst.setText(student.studentID.toString() + " " + student.studentName + System.getProperty("line.separator"))
            studentid.setText("")
            studentname.setText("")
        } else {
            lst.text = "No Match Found"
            studentid.setText("")
            studentname.setText("")
        }
    }

    fun deleteStudent(view: View) {
        val dbHandler = MyDBHandler(this,
                null, null, 1)
        val result = dbHandler.deleteHandler(Integer.parseInt(
                studentid.text.toString()))
        if (result) {
            studentid.setText("")
            studentname.setText("")
            lst.text = "Record Deleted"
        } else
            studentid.setText("No Match Found")
    }

    fun updateStudent(view: View) {
        val dbHandler = MyDBHandler(this,
                null, null, 1)
        val result = dbHandler.updateHandler(Integer.parseInt(
                studentid.text.toString()), studentname.text.toString())
        if (result) {
            studentid.setText("")
            studentname.setText("")
            lst.text = "Record Updated"
        } else
            studentid.setText("No Match Found")
    }
}
